docs/development.md

```diff
@@ -88,3 +88,27 @@ and adding dump function in your specific calculator
         dumpOvTensorInput(input,"input");
     #endif
 ```      
+# Setting log level
+
+The log level in mediapipe application can be set using GLOG_minloglevel env variable. 0 is the most verbose for debuging purposes and 3 is the least verbose with the major errors only.
+That setting is applicable both for mediapipe framework logs and also for OpenVINO inference calculator.
+Openvino Model Server evaluates the GLOG_minloglevel value and sets its own log_level based on it to one of the following values:
+    TRACE, DEBUG, INFO, ERROR
+
+Recommended development log_level setting is DEBUG as it will print out usefull information and perform additional OpenVINOInferenceCalculator calculator validation.
```

Comment:
```suggestion
Recommended development GLOG_minloglevel setting is `0` as it will print most detailed information and perform additional graph and OpenVINOInferenceCalculator calculator validation.
```